library/src/lib/menu/menu-group.component.ts

```diff
@@ -8,12 +9,10 @@ import { Component, HostBinding, ViewEncapsulation } from '@angular/core';
     templateUrl: './menu-group.component.html',
     encapsulation: ViewEncapsulation.None,
     host: {
-        '[class.fd-has-display-block]': 'true'
+        'class': 'fd-has-display-block fd-menu__group',
```

Comment:
The problem with this (if I recall correctly) is that if a user adds a class, it causes some problems. That's why we used the class.[classname] thing initially